src/redux.js

```diff
@@ -973,6 +981,35 @@ export function getIncorrectResults(state) {
   return getResultsByGrade(state, ResultsGrades.INCORRECT);
 }
 
+export function getAllResults(state) {
+  return getResultsByGrade(state, ResultsGrades.ALL);
+}
+
+// Return results data so that it looks like regular data provided to the
+// DataTable.
+export function getResultsDataInDataTableForm(state) {
+  const resultsByGrades = getAllResults(state);
+
+  if (!resultsByGrades || resultsByGrades.examples.length === 0) {
+    return null;
+  }
+
+  const numItems = Math.min(10, resultsByGrades.examples.length);
```

Comment:
Out of curiosity, why is 10 passed in here but 7 is passed in to the `getNumItems` function in `GenerateResults`?

Reply:
Great question.  I picked this arbitrarily, since none of the existing users of the code need more than 10, though you're right that they specifically don't need more than 7 at the moment.  I think I'll leave this value as-is, but add a comment explaining why.